static/app/views/performance/transactionSummary/transactionOverview/index.tsx

```diff
@@ -106,7 +109,25 @@ function OverviewContentWrapper(props: ChildProps) {
     queryExtras,
   });
 
+  const unfilteredQueryExtras = getTransactionMEPParamsIfApplicable(
+    mepSetting,
+    organization,
+    location,
+    true
+  );
+
+  const additionalQueryData = useDiscoverQuery({
+    eventView: getUnfilteredTotalsEventView(eventView, location),
+    orgSlug: organization.slug,
+    location,
+    transactionThreshold,
+    transactionThresholdMetric,
+    referrer: 'api.performance.transaction-summary',
+    queryExtras: unfilteredQueryExtras,
+  });
+
```

Comment:
This will make a query still even with the feature flag  `performance-metrics-backed-transaction-summary` not set, I don't think this is a huge deal perf wise for now, but if you want to clean this up if you plan on having this out for more then a couple days without the flag on, you can extend `useDiscoverQuery` -> `useGenericDiscoverQuery` to accept options as a second param which would allow passing react query opts through. Then you can use something like:

```js
 const additionalQueryData = useDiscoverQuery({
    eventView: getUnfilteredTotalsEventView(eventView, location),
    orgSlug: organization.slug,
    location,
    transactionThreshold,
    transactionThresholdMetric,
    referrer: 'api.performance.transaction-summary',
    queryExtras: unfilteredQueryExtras,
  }, {
    enabled: organization.features.includes('performance-metrics-backed-transaction-summary')
  });
```